InventoryBuilder.py

```diff
@@ -76,22 +76,19 @@ def iteration():
         @app.route('/iteration_store', methods=['GET'])
         def iteration_store():
             return_iteration = self.successful_iteration_list
-            return(json.dumps(return_iteration))
+            return json.dumps(return_iteration)
 
         # FIXME: this could basically be the always active token list. no active token? refresh!
         @app.route('/target_tokens', methods=['GET'])
         def token():
             return json.dumps(self.target_tokens)
 
         try:
-            if os.environ['DEBUG'] >= '2':
-                WSGIServer((self.wsgi_address, self.port), app).serve_forever()
```

Comment:
Can't we check on numeric loglevel here? I'd like to have the possibility to make WSGIServer talking.

Reply:
I will take care

Reply:
The `log` parameter in WSGiServer – If given, an object with a `write`  method to which request logs will be written. So I passed my created logger object to this. ` WSGIServer((self.wsgi_address, self.port), app, log=logger).serve_forever()`

This works perfectly fine:

```
[2020-12-07 08:44:54,892] [DEBUG] Collecting VM: 1000eyes.cc.qa-de-1.cloud.sap
[2020-12-07 08:44:54,892] [DEBUG] Collecting VM: dhcp02.cc.qa-de-1.cloud.sap
[2020-12-07 08:44:54,901] [INFO] Inventory relaxing before going to work again
[2020-12-07 08:44:59,777] [INFO] 127.0.0.1 - - [2020-12-07 08:44:59] "GET /vrops_list HTTP/1.1" 200 153 0.000400
[2020-12-07 08:44:59,817] [INFO] 127.0.0.1 - - [2020-12-07 08:44:59] "GET /target_tokens HTTP/1.1" 200 232 0.000331
[2020-12-07 08:45:04,821] [INFO] 127.0.0.1 - - [2020-12-07 08:45:04] "GET /iteration HTTP/1.1" 200 116 0.000276
[2020-12-07 08:45:04,836] [INFO] 127.0.0.1 - - [2020-12-07 08:45:04] "GET /target_tokens HTTP/1.1" 200 232 0.000335
[2020-12-07 08:45:09,846] [INFO] 127.0.0.1 - - [2020-12-07 08:45:09] "GET /iteration HTTP/1.1" 200 116 0.000392
[2020-12-07 08:45:09,863] [INFO] 127.0.0.1 - - [2020-12-07 08:45:09] "GET /target_tokens HTTP/1.1" 200 232 0.000457
[2020-12-07 08:45:14,871] [INFO] 127.0.0.1 - - [2020-12-07 08:45:14] "GET /iteration HTTP/1.1" 200 116 0.000381
[2020-12-07 08:45:14,888] [INFO] 127.0.0.1 - - [2020-12-07 08:45:14] "GET /target_tokens HTTP/1.1" 200 232 0.000325
[2020-12-07 08:45:19,895] [INFO] 127.0.0.1 - - [2020-12-07 08:45:19] "GET /iteration HTTP/1.1" 200 116 0.000395
[2020-12-07 08:45:19,907] [INFO] 127.0.0.1 - - [2020-12-07 08:45:19] "GET /target_tokens HTTP/1.1" 200 232 0.000333
```

Reply:
Due to timestamp doubling per line, I will rather leave WSGi logging standalone by checking on numeric log level as you suggested.